### ISSUE TITLE
how to get right root transition?

### ISSUE THREAD

**Issue by lucasjinreal** [open-mmlab/mmhuman3d#246]
Hi, I current converted the result to fbx animation, it looks fine on all rotations, but only the transition is not right.

![image](https://user-images.githubusercontent.com/21303438/186412714-cb37ffdb-5998-4995-b9ab-ff50c8c2c3ee.png)

![image](https://user-images.githubusercontent.com/21303438/186412741-6369adf3-74fe-4c4b-89dc-e53e25e66860.png)

I get the global_orientation which is the first bone position as root position, and set it in blender, I am not sure how to align with blender,

Does this order fit the blender order?

```
bones[bone_name_index_choosen[0]].location = (
            # Vector((100 * trans[1], 100 * trans[2], 100 * trans[0]))
            # Vector((100 * trans[0], 100 * trans[1], 100 * trans[2]))
            Vector((trans[1]*30, trans[2], pelvis_position[-1]))
        )
```

I tried also commented way all not works good. how to make it natural?

**Comment by zetyquickly** [comment]
I'm also encountered this question. One thing that I've deduced is whenever we use VIBE or PARE the `pred_cameras` are needed to be converted as it done in `visualize_smpl_vibe` flow.

VIBE and PARE outputs [N,4] valued arrays while HMR/SPIN [N,3] cameras

**Comment by zetyquickly** [comment]
Regarding the location. Have you tried `Vector((100*trans[1], 0, 100*trans[0]))`.

**Comment by jinfagang** [comment]
@zetyquickly Hi, am using PARE but got a [-1, 3] output from pred_cams, I also tried using pred_cams as transition.

But it's just looks not right. Very shaking and also some jitting on x or z axis.

How did u using pred_cams? Directly applied into blender bone position without any further transform?

**Comment by zetyquickly** [comment]
Yeah, you right it is a [N,3] tensor, but consider that these parameters are weakperspective camera parameters not an x,y,z. Follow `visualize_smpl_vibe` flow  to get `orig_camera` or `K, R, T` matrices.

**Comment by zetyquickly** [comment]
I could achieve the following performance. 3D model is well aligned but whenever the person on the video do "sit" the model do "jump" 
[tiktok source](https://www.tiktok.com/@matayasweeting/video/7045393332893404463?is_from_webapp=v1&item_id=7045393332893404463)

https://user-images.githubusercontent.com/25350960/186831431-16f886b5-6d6a-4b27-8fa9-3034bc8e0e52.mp4

**Comment by jinfagang** [comment]
@zetyquickly Can we add a wechat connection, we can have a further discussion on this topic, your transition also looks not right, but I think I can make it right,

**Comment by ttxskk** [comment]
Hi @jinfagang, 
I think you have resolved this problem after the discussion in the WeChat group. Would you mind updating it for other users?  
Thanks.

**Comment by jinfagang** [comment]
@ttxskk Honestly speaking, I forgot how did we solve this...... Would u remind me a little bit?

**Comment by zetyquickly** [comment]
@ttxskk could you share we chat qr code?

**Comment by ttxskk** [comment]
> 

[Here](https://github.com/open-mmlab/mmhuman3d/blob/main/README_CN.md) you can find the QR code.

**Comment by ttxskk** [comment]
> @ttxskk could you share we chat qr code?

[Here](https://github.com/open-mmlab/mmhuman3d/blob/main/README_CN.md) you can find the QR code.

**Comment by zhewei-mt** [comment]
Hello @jinfagang ，I have the same question about the translation in real world space and thanks for posting this question. Do you succeed in resolving this question? 
Would you mind establishing a wechat connection? If possible, please leave a way to reach out your wechat or you can add my wechat via 13989773123. Any hint will be appreciate!!

**Comment by bbbbubble** [comment]
@jinfagang @ttxskk how did you solve this problem, and could you share some comparison regarding this issue? Because i'm not sure what the problem is?

**Comment by bbbbubble** [comment]
@jinfagang @ttxskk I have encountered the same problem. Could you share how to deal with this issue, or some insights? I can't enter the wechat group even though I have scanned the QR code...

**Comment by ttxskk** [comment]
Hi @bbbbubble,

You can refer to this issue #262. Please let me know if you have any other questions.
